src/Sulu/Bundle/CategoryBundle/Api/Category.php

```diff
@@ -348,6 +356,17 @@ public function getTranslation()
                 return $translation;
             }
         }
+
+        return;
+    }
+
+    /**
+     * Creates a new Translation for category.
+     *
+     * @return CategoryTranslation
+     */
+    public function createTranslation()
```

Comment:
hmm the `getTranslation` was public and not to make a to big BC we should also make this public ... i think!